package.json

```diff
@@ -1,18 +1,28 @@
 {
   "homepage": "https://aladin.u-strasbg.fr/",
   "name": "aladin-lite",
+  "type": "module",
   "version": "3.1.0",
   "description": "An astronomical HiPS visualizer in the browser",
   "author": "Thomas Boch and Matthieu Baumann",
   "license": "GPL-3",
-  "main": "dist/aladin.js",
+  "module": "./dist/aladin.js",
+  "main": "./dist/aladin.js",
   "files": [
-    "dist/*"
+    "dist/aladin.js",
+    "dist/aladin.umd.cjs",
+    "dist/index.html"
   ],
+  "exports": {
+    ".": {
+      "import": "./dist/aladin.js"
```

Comment:
@bmatthieu3 you could add a second option to use umd file by default as mentioned in [conditional exports](https://nodejs.org/api/packages.html#conditional-exports).
`"require": "./dist/aladin.umd.cjs"`

Reply:
done, do not hesitate if there are other things